cws-engine-service/src/main/java/jpl/cws/engine/EngineDbService.java

```diff
@@ -144,64 +148,76 @@ public void updateStats(File[] logs, long diskFreeBytes) {
 	 */
 	public void createOrUpdateWorkerRow(String lockOwner) {
 		if (!workerExists()) {
-			//
-			// ROW FOR WORKER DOES NOT EXIST, SO CREATE ONE
-			//
+			if ( cwsInstallType.equals("console_only") && workerConsoleOnlyTypeExists() ) {
+				//
+				// ROW FOR CONSOLE ONLY WORKER ALREADY EXISTS
+				//
+				log.info("Worker Row with cws_install_type: " + cwsInstallType + ", already exists. ");
```

Comment:
Should this be an error?

Reply:
Let's actually make this a warning
```suggestion
				log.warn("Worker Row with cws_install_type: " + cwsInstallType + ", already exists. ");
```

Reply:
@ztaylor54  
Yes, changed that. Thanks

Reply:
or an "error"